crypto/evp/p_lib.c

```diff
@@ -205,10 +205,14 @@ EVP_PKEY *EVP_PKEY_new(void)
     return ret;
 }
 
-void EVP_PKEY_up_ref(EVP_PKEY *pkey)
+int EVP_PKEY_up_ref(EVP_PKEY *pkey)
 {
     int i;
     CRYPTO_atomic_add(&pkey->references, 1, &i, pkey->lock);
```

Comment:
Are there currently any _up_ref() with the int return that don't check for errors? I think I already converted them all during the threading conversion (though I guess I may have missed some).